source/administration/config-settings.rst

```diff
@@ -3953,6 +3988,20 @@ Changes made when hardened mode is enabled:
 | This feature’s ``config.json`` setting is ``"ExperimentalEnableHardenedMode": false`` with options ``true`` and ``false`` for above settings respectively.          |
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Ldap Group Sync (Experimental)
```

Comment:
Propose `Ldap` --> `AD/LDAP` for consistency with other user-facing strings. Would double check elsewhere in the doc, though I think this was the only place `Ldap` was used.